StructureDefinition/Extension-England-COVID19TestResult.xml

```diff
@@ -58,41 +42,45 @@
       </slicing>
       <min value="0" />
```

Comment:
This minimum cardinality of 0 was not part of the original extension you worked on. 
[ExtensionCOVID19TestResult](https://simplifier.net/nhsdigital/68172784-12f2-4437-8680-2595272a885f)